app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class RecipesController < ApplicationController
+  def show
+    @recipe = Recipe.find(params[:id])
+    @total_cost = @recipe.ingredients.sum(:cost)
```

Comment:
Same thing here, for total cost. We will want to create that `total_cost` method in our `recipe` model and then instead of calling that method here in our controller we will call that method in our view.